brainframe/cli/translations/update.en.yml

```diff
@@ -13,14 +13,15 @@ en:
   complete: "BrainFrame has been updated!"
   upgrade-version: "BrainFrame:%{existing_version} detected under your installation
   location, we will replace it with BrainFrame:%{requested_version}"
+  version-failing: "You have BrainFrame:%{existing_version} installed already.
+  The latest version we found on updating server is
```

Comment:
> on updating server

what does this mean?

Reply:
I updated the wording to make it more clear.